api/read.rb

```diff
@@ -0,0 +1,4 @@
+post "#{APIPREFIX}/users/:user_id/read" do |user_id|
```

Comment:
Hi @wajeeha-khalid.  Can you explain what made you use "users" in the base of this endpoint, rather than "threads"?  Also, why did you add this to a new file vs in users.rb?  Thanks.